src/Symfony/Bridge/Twig/Tests/Extension/DumpExtensionTest.php

```diff
@@ -96,14 +96,14 @@ public static function getDumpArgs()
             [
                 [],
                 [123, 456],
-                "<pre class=sf-dump id=sf-dump data-indent-pad=\"  \"><span class=sf-dump-num>123</span>\n</pre><script>Sfdump(\"sf-dump\")</script>\n"
-                ."<pre class=sf-dump id=sf-dump data-indent-pad=\"  \"><span class=sf-dump-num>456</span>\n</pre><script>Sfdump(\"sf-dump\")</script>\n",
+                "<pre class=sf-dump id=sf-dump data-indent-pad=\"  \"><span class=\"sf-dump-num\">123</span>\n</pre><script>Sfdump(\"sf-dump\")</script>\n"
+                ."<pre class=sf-dump id=sf-dump data-indent-pad=\"  \"><span class=\"sf-dump-num\">456</span>\n</pre><script>Sfdump(\"sf-dump\")</script>\n",
```

Comment:
this change breaks low-deps
better preserve the previous output when possible to not have to change version constraints
doable?

Reply:
I think I could quote the `class` attribute only if there are two indeed (would feel weird though :sweat_smile:)

Reply:
that'd help fix tests (they're still red :) )

Reply:
Wouldn’t high and low-deps tests eventually pass when the code is released?

Anyways, I limited the changes to ellipsization so that other components’ tests are not impacted :green_circle: